### ISSUE TITLE
Move constants to params

### ISSUE THREAD

**Issue by alpe** [CosmWasm/wasmd#202]
## Summary
We have a couple of hard coded values that could be moved into params so that we can update them without deployment. 
An incomplete list:

- max contract size
- gas multiplier
- instance cost
- compile cost
...

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by ethanfrey** [comment]
Great ideas. Use the current constants as defaults and allow governance to adjust it.

**Comment by ethanfrey** [comment]
Note: I will have to implement https://github.com/CosmWasm/go-cosmwasm/issues/122 before it is safe to change GasMultiplier. Let's wait on this until I update go-cosmwasm

**Comment by alpe** [comment]
We have closed the PR and postpone this feature

**Comment by ethanfrey** [comment]
This is related to #809 

We need some of these values in the stateless ValidateBasic functions. That will make usage of Params impossible.

As a first step, I would propose pulling them into module-level `var` rather than `const`. 
Projects importing `x/wasm` could then customize them either by [runtime flags](https://github.com/CosmWasm/wasmd#runtime-flags) or by setting them explicitly in their custom `app.go` or `cmd/*/main.go`

**Comment by alpe** [comment]
Closing this now as this was done with the gas register. MaxLabelSize, MaxWasmSize, MaxProposalWasmSize are vars in `types/validation.go`
